showcase/scripts/generate_showcase.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Generates showcase without post processing (i.e. raw gapic, grpc and proto
+# libraries). It will compute the showcase version from
+# `sdk-platform-java/showcase/gapic-showcase/pom.xml`. The generator version is
+# inferred from versions.txt
+set -ex
+
+readonly SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+lib_gen_scripts_dir="${SCRIPT_DIR}/../../library_generation/"
+source "${lib_gen_scripts_dir}/utilities.sh"
+readonly perform_cleanup=$1
+
+cd "${SCRIPT_DIR}"
+
+# clone gapic-showcase
+if [ ! -d schema ]; then
+  if [ -d gapic-showcase ]; then
+    rm -rdf gapic-showcase
+  fi
+  # looks at sdk-platform-java/showcase/gapic-showcase/pom.xml to extract the
+  # version of gapic-showcase
+  # see https://github.com/googleapis/gapic-showcase/releases
+  showcase_version=$(xmllint --xpath '/*[local-name()="project"]/*[local-name()="properties"]/*[local-name()="gapic-showcase.version"]/text()' "${SCRIPT_DIR}/../gapic-showcase/pom.xml")
```

Comment:
We have to install `xmllint` before running the script.

Do you think retrieving the version using `sed` and `cut` would be better?

Reply:
I'll try :)